Cargo.toml

```diff
@@ -8,6 +8,6 @@ license = "MPL-2.0"
 repository = "https://github.com/neXromancers/uniconize"
 
 [dependencies.x11rb]
-version = "0.4"
+version = "0.5"
+# Disable the "allow-unsafe-code" feature and the libxcb dependency
```

Comment:
I guess this does deserve being documented inline and not just in the commit log. :+1: